ddl/schema_test.go

```diff
@@ -133,7 +136,55 @@ func testCheckJobCancelled(c *C, d *ddl, job *model.Job) {
 	})
 }
 
-func (s *testSchemaSuite) TestSchemaT(c *C) {
+func getSchemaVer(c *C, ctx context.Context) int64 {
+	txn, err := ctx.GetTxn(true)
+	c.Assert(err, IsNil)
+	c.Assert(txn, NotNil)
+	m := meta.NewMeta(txn)
+	ver, err := m.GetSchemaVersion()
+	c.Assert(err, IsNil)
+	return ver
+}
+
+type historyJobArgs struct {
+	ver    int64
+	tbl    *model.TableInfo
+	tblIDs map[int64]struct{}
+}
+
+func checkHistoryJobArgs(c *C, ctx context.Context, id int64, args *historyJobArgs) error {
```

Comment:
The return value seems always be nil?
Can the return value declaration of this function be removed.